whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/poem/PoeticFormHasOnlyDbmMaintainerValidator.java

```diff
@@ -0,0 +1,44 @@
+package net.ripe.db.whois.update.handler.validator.poem;
+
+import com.google.common.collect.ImmutableList;
+import net.ripe.db.whois.common.domain.CIString;
+import net.ripe.db.whois.common.rpsl.AttributeType;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslAttribute;
+import net.ripe.db.whois.update.domain.Action;
+import net.ripe.db.whois.update.domain.PreparedUpdate;
+import net.ripe.db.whois.update.domain.UpdateContext;
+import net.ripe.db.whois.update.domain.UpdateMessages;
+import net.ripe.db.whois.update.handler.validator.BusinessRuleValidator;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+import static net.ripe.db.whois.common.domain.CIString.ciString;
+
+@Component
+public class PoeticFormHasOnlyDbmMaintainerValidator implements BusinessRuleValidator {
+
+    private static final ImmutableList<Action> ACTIONS = ImmutableList.of(Action.CREATE, Action.MODIFY);
+    private static final ImmutableList<ObjectType> TYPES = ImmutableList.of(ObjectType.POETIC_FORM);
+
+    private static final CIString POETC_FORM_MAINTAINER = ciString("RIPE-DBM-MNT");
+
+    @Override
+    public void validate(final PreparedUpdate update, final UpdateContext updateContext) {
+        final List<RpslAttribute> mntByAttribute = update.getUpdatedObject().findAttributes(AttributeType.MNT_BY);
```

Comment:
update.getUpdatedObject().getValuesForAttribute(AttributeType.MNT_BY); is a bit cleaner?

Reply:
updateContext.addMessage expects RpslAttribute and this will return CIstring. Also with set I need to then use iterator.

Reply:
OK! Thanks for explaining